pulp_rpm/app/tasks/copy.py

```diff
@@ -24,12 +32,24 @@ def find_children_of_content(content, repository_version):
     # All packages in the source repository version
     package_ids = repository_version.content.filter(
         pulp_type=Package.get_pulp_type()).only('pk')
+    # All packagecategory in the source repository version
+    packagecategory_ids = repository_version.content.filter(
+        pulp_type=PackageCategory.get_pulp_type()).only('pk')
+    # All packagecategories in the source repository version
+    packageenvironment_ids = repository_version.content.filter(
+        pulp_type=PackageEnvironment.get_pulp_type()).only('pk')
+    # All packagegroups in the source repository version
+    packagegroup_ids = repository_version.content.filter(
+        pulp_type=PackageGroup.get_pulp_type()).only('pk')
     # All modules in the source repository version
     module_ids = repository_version.content.filter(
         pulp_type=Modulemd.get_pulp_type()).only('pk')
```

Comment:
@dralley , @pavelpicka , does it make sense to have comments for every query here?
If the concern is that it's not clear that the `repository_version` is a source repository version, should we rename it to `src_repo_version`?

Reply:
Liked the idea of 'src_repo_version', so refactored. 
I reduced comments but good to keep them there as two types of queries are present. Source repo content and content to copy.